recipes/cuda-api-wrappers/all/conandata.yml

```
@@ -1,7 +1,23 @@
 sources:
+  "0.7.0-b2":
+    url: "https://github.com/eyalroz/cuda-api-wrappers/archive/0.7.0-b2.tar.gz"
+    sha256: "9439cb2250dd3045a05d43c4ca66b5d49535eeba123b05a2e49169354fdb3123"
   "0.7-b1":
     url: "https://github.com/eyalroz/cuda-api-wrappers/archive/0.7b1.tar.gz"
     sha256: "1ed5912d8f602ccd176865b824de17f462cb57142eb2a685d7cc034831e54a71"
+  "0.6.8":
+    url: "https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v0.6.8.tar.gz"
+    sha256: "a0d1b062dbe41c99d06df4ae7885a053c2ae3815d6fe12df0458bc5277d08ed7"
+  "0.6.7":
+    url: "https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v0.6.7.tar.gz"
+    sha256: "e00f0ac492fde5c39211c69d3c8ae8c694d74cb6603924be91609eb9f7c4c722"
+  "0.6.6":
+    url: "https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v0.6.6.tar.gz"
+    sha256: "271e7393725bc42feaf1d58418ea6292fdfbe5814070c590ffaabdacc3c73e8f"
+  "0.6.4":
+    url: "https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v0.6.4.tar.gz"
+    sha256: "7ef7de5ae82b42d67bae346e49863ee610f387cccb65c2cb8edc46ff8f3b3342
```

```suggestion
  "0.6.8":
    url: "https://github.com/eyalroz/cuda-api-wrappers/archive/refs/tags/v0.6.8.tar.gz"
    sha256: "a0d1b062dbe41c99d06df4ae7885a053c2ae3815d6fe12df0458bc5277d08ed7"
```
The convention on CCI is to add intermediate versions only when there's a specific need for them. The latest version is probably sufficient.

---

Ok. But I will keep the added beta, to mask the beta which should never have been added in the first place.